### PR TITLE
Chore: disable dialog auto resize #128

### LINKED ISSUE #128: Dialog auto-resize should be disabled
## Description

The dialog Electorn auto-resize feature should be disabled. Auto-resizing is handled internally by ElectronIM and should not be delegated to Electron's built-in behavior.

The following code:
https://github.com/manusa/electronim/blob/bcb5760d80dd00ec51055e1c404c02ff55eed874/src/browser-window/index.js#L20

Should be replaced by:
```javascript
  browserView.setAutoResize({width: false, horizontal: false, height: false, vertical: false});
```

The test should be updated accordingly too:
https://github.com/manusa/electronim/blob/bcb5760d80dd00ec51055e1c404c02ff55eed874/src/browser-window/__tests/index.test.js#L39-L40

### PR DESCRIPTION
**Pull request by Aahil13**
Fix #128

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,10 +2,7 @@
   "name": "electronim",
   "version": "0.0.0",
   "description": "Free/Libre open source Electron based multi instant messaging (IM) client.",
-  "author": {
-    "name": "Marc Nuri",
-    "url": "https://www.marcnuri.com"
-  },
+  "author": "Marc Nuri (https://www.marcnuri.com)",
```
Comment (manusa) [inline package.json:5]: Please restore the changes to the package.json or send **only** the one for the `directories` entry in a different commit.

**Review by manusa — CHANGES_REQUESTED** [review]
LGTM except for the format issues, please fix and we'll be good to go.

File: package.json (hunk 1 above)
Comment (Aahil13) [reply package.json:5]: Noted. I will work on this ASAP

Comment (Aahil13) [reply package.json:5]: I have made the necessary changes to the package.json file, as well as the formatting on the index.js and index.test.js files

**Review by manusa — APPROVED** [review]

**Pull request merged**